assets/css/metabox-classic.css

```diff
@@ -0,0 +1,12 @@
+/**
+ * Metabox Tabs
```

Comment:
We are using `LESS` css in our plugins, manual CSS codes aren't allowed. Move all the CSS codes to a LESS file and do it from there.